### ISSUE TITLE
[cleanup] Move trap signals to Main

### ISSUE THREAD

**Issue by harshavardhana** [minio/minfs#4]
```
        // todo(nl5887): move trap signals to Main, this is not supposed to be in Serve
        signalCh := make(chan os.Signal, 1)
        signal.Notify(signalCh, os.Interrupt, os.Kill, syscall.SIGUSR1)
```

**Comment by rushenn** [comment]
ETA:2017-06-06 12:00

**Comment by rushenn** [comment]
ETA: 2017-06-14 12:00

**Comment by harshavardhana** [comment]
Fixed by #55
